kernel/indTyping.ml

```diff
@@ -70,6 +70,9 @@ type univ_info = { ind_squashed : bool; ind_has_relevant_arg : bool;
                    missing : Sorts.t list; (* missing u <= ind_univ constraints *)
                  }
 
+let no_sort_variable () =
```

Comment:
I'm a big confused about the name here. The error says a sort was sent, but the name says no sort?

Reply:
The name means that there *should* be no sort in the kernel.